### DISCUSSION TITLE
Difficulty with spaCy Model Path in Azure ML VS Code

### DISCUSSION THREAD

**Discussion by ul-sherin** [explosion/spaCy#13165] [Help: Installation]
I'm currently facing an issue while working with Azure ML VS Code and spaCy. Here's the problem I'm encountering:
 
OSError: [E050] Can't find model 'en_core_web_lg'. It doesn't seem to be a Python package or a valid path to a data directory.

I am attempting to convert a JSON file to spaCy, and I've installed spaCy for this purpose. However, I'm unable to resolve the mentioned error. I've come across suggestions in a blog recommending providing the exact path for 'en_core_web_lg'. Unfortunately, in the Azure ML VS Code environment, the file is not saved locally, making it challenging to identify its actual path.
 
Is there a workaround for specifying the path of 'en_core_web_lg' without having it saved locally?

**Comment by adrianeboyd** [comment]
`en_core_web_lg` isn't required by default by spacy, so are you using another library for your conversion?

You can either install it with `spacy download en_core_web_lg` or add it to your requirements with the direct URL:

Get the URL:

```
spacy info en_core_web_lg --url
```

Add to `requirements.txt`:

```
https://github.com/explosion/spacy-models/releases/download/en_core_web_lg-3.7.1/en_core_web_lg-3.7.1-py3-none-any.whl
```

The exact URL will depend on the version of spacy that you want to use.
